index.html

```diff
@@ -7,15 +7,61 @@
     <link href='//fonts.googleapis.com/css?family=Roboto:400,500,300' rel='stylesheet' type='text/css'>
     <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
     <link rel="stylesheet" href="css/normalize.css">
-    <!-- Add a link to your CSS file here (use the line above to guide you) -->
     <link rel="stylesheet" href="css/style.css">
     <link rel="shortcut icon" type="image/x-icon"  href="favicon.ico">
 </head>
+
 <body>
+    <header class="header">
+        <img class="header-img" src="img/karma-logo.svg" alt="Karma logo" width="26">
+        <nav class="navigation">
+            <ul class="links">
+                <li class="active"><a href=" ">Meet Karma</a></li>
+                <li><a href="">How it Works</li>
+                <li><a href="">Store</a></li>
+                <li><a href="">Blog</a></li>
+                <li><a href="">Help</a></li>
+                <li><a href="">Login</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section class="karma-introduction">
+            <h1>Introducing Karma</h1>
+
+            <h2>Bring WiFi with you, everywhere you go.</h2>
+
+            <button type="button">Learn More</button>
+        </section>
+
+        <section class="about">
+            <h1>Everyone needs a little Karma.</h1>
+
+            <ul class="benefits">
+                <li><img width="127" height="127" src="img/icon-devices.svg" alt="Laptop, tablet and phone icon">
```

Comment:
Alt text here might not be useful as we have the text with the description anyway. Generally it is really good to have though!

[There's a good article here](https://blog.hubspot.com/marketing/image-alt-text) giving good and bad examples.